how-to-organize-a-meetup.md

```diff
@@ -0,0 +1,133 @@
+# How to plan a meetup
+
+This document is meant to help you plan a meetup. It is specifically tailed for the IPFS community; however, in order to make it more general, I have put every IPFS-only task into its own section.
+
+## Pre-planning
+
+Before hosting a meetup, you should identify: 
+
+- Who is the meetup for? Beginners? Advanced users? Advanced Developers?
+- What topics should be covered?
+- What are your core values, and how do you want these to come off? Inclusiveness;
```

Comment:
Why does this say `Inclusiveness;` at the end of the line? Did you mean to add a note about inclusiveness? That would be a good idea. Otherwise, why is it there?